cognite/v06/dto.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""Data Objects
+
+This module contains data objects used to represent the data returned from the API.
+"""
+
+from typing import List, Union
+
+
+class Column:
+    """Data transfer object for a column.
+
+    Args:
+        id (int):           ID of the column.
+        name (str):         Name of the column.
+        external_id (str):  External ID of the column.
+        data_type (str):    Data type of the column.
+        metadata (dict):    Custom, application specific metadata. String key -> String Value.
+    """
+
+    id: Union[int, None]
+    name: str
+    external_id: str
+    data_type: str
+    metadata: dict
+
+    def __init__(
+            self,
+            id: Union[int, None],
+            name: str,
+            external_id: str,
+            data_type: str,
+            metadata: dict
+    ):
+        self.id = id
+        self.name = name
+        self.external_id = external_id
+        self.data_type = data_type
+        self.metadata = metadata
+
+
+class Sequence:
+    """Data transfer object for a sequence.
+
+    Args:
+        id (int):           ID of the sequence.
+        name (str):         Name of the sequence.
+        external_id (str):  External ID of the sequence.
+        asset_id (int):     ID of the asset the sequence is connected to, if any.
+        columns (list):     List of columns in the sequence.
+        description (str):  Description of the sequence.
+        metadata (dict):    Custom, application specific metadata. String key -> String Value.
+    """
+
+    id: Union[int, None]
+    name: str
+    external_id: str
+    asset_id: int
+    columns: List[Column]
+    description: str
+    metadata: dict
+
+    def __init__(
+            self,
+            id: Union[int, None],
+            name: str,
+            external_id: str,
+            asset_id: Union[int, None],
+            columns: List[Column],
+            description: str,
+            metadata: dict
+    ):
+        self.id = id
```

Comment:
I usually name the attributes with camelCase so that when i do object.__dict__, it returns the json object correctly formatted.